src/app/recover-funds/[address]/RecoverFundsButton.tsx

```diff
@@ -25,8 +39,11 @@ function RecoverFundsButton({
 }) {
   const [message, setMessage] = useState('');
   const [loading, setLoading] = useState(false);
+  const { isSmartContractWallet } = useAccountType();
```

Comment:
We only use this flag to show the popup "To continue, please approve tx on your smart contract wallet.", by this time, user had already made a transaction, so most likely the loading has been completed, but I added the loading flag just in case